code/modules/vehicles/unmanned/unmanned_vehicle_remote.dm

```diff
@@ -3,6 +3,7 @@
 	desc = "Used to control an unmanned vehicle.<br>Tap the vehicle you want to control with the controller to link it."
 	icon = 'icons/obj/device.dmi'
 	icon_state = "multitool2"
+	w_class = WEIGHT_CLASS_TINY
```

Comment:
why's it tiny? Everything else like this is small

Reply:
thought it would be nice to have it on helmet but i'll change it